app/printTemplates/summary/local/summary.php

```diff
@@ -0,0 +1,69 @@
+<?php
```

Comment:
This is the default summary (not the object one). Same contents as `local/ca_objects_summary.php` but without the different image handling.